CHANGELOG.md

```diff
@@ -4,6 +4,57 @@
 
 <!-- next version -->
 
+## v0.9.0
+
+This release includes version 0.104.0 of the upstream Collector components.
+
+The individual upstream Collector changelogs can be found here:
+
+v0.104.0:
+
+- <https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.104.0>
+- <https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.104.0>
+
+<details>
+<summary>Highlights from the upstream Collector changelog</summary>
+</br>
+
+### 🛑 Breaking changes 🛑
+
+This release includes 2 very important breaking changes.
+
+1. The `otlpreceiver` will now use `localhost` by default instead of `0.0.0.0`. This may break the receiver in containerized environments like Kubernetes. If you depend on `0.0.0.0` disable the `component.UseLocalHostAsDefaultHost` feature gate or explicitly set the endpoint to `0.0.0.0`.
```

Comment:
```suggestion
1. The `otlpreceiver` will now use `localhost` by default instead of `0.0.0.0`. This may break the receiver when the sender is not running on the same host, particularly in containerized environments like Kubernetes. If you depend on `0.0.0.0` disable the `component.UseLocalHostAsDefaultHost` feature gate or explicitly set the endpoint to `0.0.0.0`.
```